### ISSUE TITLE
browser window and body text frame hiccup

### ISSUE THREAD

**Issue by stationaery** [techworkersco/techworkersco.github.io#67]
When the browser window is <1075px and >825px, the body text frame pinches to a much more narrow width than even <825px and >635px (at which point when the menu serves up a hamburger ≡). I'm not seeing anything in the CSS file; might be something someone knows how to fix in Bootstrap?

**Comment by jessesquires** [comment]
The hamburger menu is expected behavior.

Is the issue that the column of text is too narrow?

**Comment by stationaery** [comment]
Correct- it's an issue with the body text width at a certain window width
interval.

Nothing to do with the burger, which is fine! I mentioned it only to
indicate the point at which the text hiccup occurs.

**Comment by jessesquires** [comment]
cool. yeah we can tweak that.

docs: https://getbootstrap.com/docs/4.5/layout/grid/#grid-options

**Comment by stationaery** [comment]
FWIW, here's a 43sec screencast recording showing the text frame width hiccup in action: https://drive.google.com/open?id=17OdL2dIVwE8HtF04a4vXOuSVG5ysLBPw
